src/components/organisms/FileTreePane/FileTreePane.tsx

```diff
@@ -57,9 +58,12 @@ const createNode = (
   rootFolderName: string
 ): TreeNode => {
   const resources = getResourcesForPath(fileEntry.filePath, resourceMap);
+  const isRoot = fileEntry.name === ROOT_FILE_ENTRY;
+  const key = isRoot ? ROOT_FILE_ENTRY : fileEntry.filePath;
+  const name = isRoot ? rootFolderName : fileEntry.name;
 
   const node: TreeNode = {
-    key: fileEntry.filePath,
```

Comment:
This caused an inconsistency between the tree key and the file map key which gave subtle bugs. I made the keys between both consistent.